src/DssSpell.sol

```diff
@@ -176,10 +185,25 @@ contract DssSpellAction is DssAction {
         setIlkStabilityFee(YFI_ILK, FIVE_PT_FIVE_PCT, true);
 
         // Interim DAO Budget (Note: we are leaving daiJoin hoped from the Pause Proxy for future payments)
-        address MCD_JOIN_DAI = getChangelogAddress("MCD_JOIN_DAI");
-        VatAbstract(vat()).suck(vow(), address(this), 100_000 * RAD);
+        // Sending 100,001 DAI and testing the return feature of the multisig for governance to retrieve the funds if necessary
```

Comment:
I don't think the weekly exec should be a testing ground. I think we can write a test that simulates this.

Reply:
We can't because only the pause proxy can execute it. I could just remove it as it appears to be passing in the fork.

Reply:
Per discussion, elsewhere, maybe update the comment here to be more descriptive of the purpose. i.e. Demonstrate the ability of governance to remove the funds from the multisig.

Reply:
The other way to do it is adding a test spell which is executed after this one. But probably is adding too much work.